src/rdsk.h

```diff
@@ -0,0 +1,45 @@
+/** @file
```

Comment:
Please refer to my notes above on headings.

Reply:
Fixed.